### PR TITLE
Added new API version 2022-03-01 to the Mobile Network service

### PR DESCRIPTION
**Pull request by satravi**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [X] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [X] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    - [X] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: custom-words.txt [hunk 1]
```diff
@@ -2484,6 +2484,22 @@ loadtest
...
+Preemptable
+Ambr
+AMBR
+rfsp
+Rfsp
+snssai
+Snssai
+NSSAI
+Tbps
+Vuln
```
Comment (sathiyan-sivathas) [inline custom-words.txt:2502]: I think these are already in this list a few lines below so you can remove them.

**Review by sathiyan-sivathas — APPROVED** [review]
Approved. One minor thing to fix.

File: custom-words.txt (hunk 1 above)
Comment (satravi) [reply custom-words.txt]: Addressed.

**Comment by raosuhas** [comment]
This PR is not ready for review by ARM .. .Please note that the step 3 in the checklist is not fulfilled: 

Adding a new API version
-[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).


Specifically , if you are adding a new api version then you must add the previous version as the first commit in the new folder , so that we are able to diff the changes between the versions. I have added "ARMChangesRequested" Tag , please remove the tag once you have fixed this issue.

**Comment by satravi** [comment]
> This PR is not ready for review by ARM .. .Please note that the step 3 in the checklist is not fulfilled:
> 
> Adding a new API version -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).
> 
> Specifically , if you are adding a new api version then you must add the previous version as the first commit in the new folder , so that we are able to diff the changes between the versions. I have added "ARMChangesRequested" Tag , please remove the tag once you have fixed this issue.

@raosuhas, Yes we have followed the manual process for new API version - https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version

Also the diff between the previous version 2022-01-01 and new version 2022-03-01 can be found in - https://github.com/Azure/azure-rest-api-specs/pull/18234/commits/a180e7ba91205a6ca632cd85fc6deeaf681e5931

**Comment by raosuhas** [comment]
This is incorrect , you need to update the new folder with the older api verison not the other way round I do not have any way to track latest changes made after the first update now. 

From the link you pasted :
Please note that it is time-consuming to review the file line by line for every api-version. When you are creating the swagger specification for the new api-version, please copy the swagger specification from the previous version into the new api-versioned folder and commit it. After that overwrite it with the changes for the new api-version. This makes it easy for us to review the changes.

Can you please fix and let me know again by removing the tag ?



---
In reply to: [1067575578](https://github.com/Azure/azure-rest-api-specs/pull/18234#issuecomment-1067575578) [](http://example.com/codeflow?ancestors=1067575578,1067575578)

**Comment by satravi** [comment]
> This is incorrect , you need to update the new folder with the older api verison not the other way round I do not have any way to track latest changes made after the first update now.
> 
> From the link you pasted : Please note that it is time-consuming to review the file line by line for every api-version. When you are creating the swagger specification for the new api-version, please copy the swagger specification from the previous version into the new api-versioned folder and commit it. After that overwrite it with the changes for the new api-version. This makes it easy for us to review the changes.
> 
> Can you please fix and let me know again by removing the tag ?
> 
> In reply to: [1067575578](https://github.com/Azure/azure-rest-api-specs/pull/18234#issuecomment-1067575578)

@raosuhas , 
- We have reverted the Mobile Network API version 2022-03-01-preview
- Added a new Mobile Network API version 2022-03-01-preview with previous version specs and examples
- Updated the README files to refer the latest version
- Updated the API version 2022-03-01-preview specs and examples with the latest changes - https://github.com/Azure/azure-rest-api-specs/pull/18234/commits/439e59dd4027b62f05cd79ddb225f048d84ef40f

**Comment by rainbowFi** [comment]
@raosuhas As mentioned in comment above, this new API version went through ARM review in [PR 6496](https://github.com/Azure/azure-rest-api-specs-pr/pull/6496) when we merged it to RPSaaSMaster. Is there some way we can mark this so that you don't spend time doing a full review of something that has already been fully reviewed by the ARM review board?

**Comment by satravi** [comment]
Thank you @raosuhas.
@raych1 Could you please help us on merging this PR?

**Comment by apalanichamy1981** [comment]
Approved

**Review by apalanichamy1981 — APPROVED** [review]
Approved

**Pull request merged**